jenkins-infra-data/generate-infra-data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# generate-infra-data.sh: Generate a JSON report named $1 in the directory $2 (with a optional version $3).
+#  Note: This script orchestrate generation by "parts": there are sub-scripts per services which generate partial content.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+REPORT_NAME="$1"
+test -n "${REPORT_NAME}"
+DIST_DIR="$2"
+test -n "${DIST_DIR}"
+mkdir -p "${DIST_DIR}"
+# Sub scripts need this
+export DIST_DIR
+VERSION="${3:-v1}"
```

Comment:
@dduportal since the data hasn't changed, I'm not sure why the public API/interface version has been bumped.

Is there any particular reason behind it?